# Generated by Django 4.0 on 2022-01-02 19:31

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0003_follow_a follow object with the same following and followed user can not be created'),
    ]

    operations = [
        migrations.AddConstraint(
            model_name='follow',
            constraint=models.UniqueConstraint(fields=('followed_user', 'following_user'), name="Can't follow the same user more than once"),
        ),
    ]
